<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPlatformPart</class>
 <widget class="QWidget" name="AddPlatformPart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add platform part</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="typeGroupBox">
     <property name="title">
      <string>Part type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="sensorRadioButton">
        <property name="text">
         <string>Sensor</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string>typeButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="weaponRadioButton">
        <property name="text">
         <string>Weapon</string>
        </property>
        <attribute name="buttonGroup">
         <string>typeButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="commRadioButton">
        <property name="text">
         <string>Comm</string>
        </property>
        <attribute name="buttonGroup">
         <string>typeButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="processorRadioButton">
        <property name="text">
         <string>Processor</string>
        </property>
        <attribute name="buttonGroup">
         <string>typeButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="moverRadioButton">
        <property name="text">
         <string>Mover</string>
        </property>
        <attribute name="buttonGroup">
         <string>typeButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="fuelRadioButton">
        <property name="text">
         <string>Fuel</string>
        </property>
        <attribute name="buttonGroup">
         <string>typeButtonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="detailsWidget" native="true">
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="specPartTypeLabel">
        <property name="text">
         <string>Specific part type:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="specPartTypeComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="partNameLabel">
        <property name="text">
         <string>Part name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="partNameLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="typeButtonGroup"/>
 </buttongroups>
</ui>
